pettingzoo/butterfly/pistonball/pistonball.py

```diff
@@ -463,8 +462,8 @@ def reset(self, seed=None, options=None):
                 -6 * math.pi, 6 * math.pi
             )
 
-        self.lastX = int(self.ball.position[0] - self.ball_radius)
-        self.distance = self.lastX - self.wall_width
+        self.ball_prev_pos = self._get_ball_position()
+        self.distance_to_wall_at_game_start = self.ball_prev_pos - self.wall_width
```

Comment:
```suggestion
        self.initial_wall_distance = self.ball_prev_pos - self.wall_width
```

Maybe a bit more concise?

Reply:
Mhm that is more concise. Though, it may be harder for a reader to differentiate whether "initial" refers to the start of a time-step or the start of the game.